impl/httpMetricsManager.go

```diff
@@ -30,10 +68,36 @@ func (m *HttpMetricsManager) GetThroughput() int {
 	return m.requestSent
 }
 
+// return the sampletime(duration) at input percentile
+func (m *HttpMetricsManager) GetDurationAt(thre int) time.Duration {
+	if !sort.IsSorted(m.records) {
+		sort.Sort(m.records)
+	}
+	queryIndx := len(m.records) * thre / 100
+	return m.records[queryIndx].duration / time.Millisecond
+}
+func (m *HttpMetricsManager) GetMeanDuration() int {
+	sum := m.durationSum.sum
+	for i := m.durationSum.index + 1; i < len(m.records); i++ {
+		sum += m.records[i].duration / time.Millisecond
+	}
+	m.durationSum.index = len(m.records)
+	m.durationSum.sum = sum
+	return int(sum) / len(m.records)
+}
+
+// Data for each response shoud be periodically dump to disk
+func (m *HttpMetricsManager) DumpToDisk() {
```

Comment:
this should not be needed, as users will define `PostProcessor` in `ChukonuRequest`, and there they will specify if they need to dump the response